integration/airflow/openlineage/airflow/macros.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from openlineage.airflow.utils import JobIdMapping, openlineage_job_name, safe_import_airflow
+
+_JOB_NAMESPACE = os.getenv('OPENLINEAGE_NAMESPACE', 'default')
+
+
+def get_create_session():
+    return safe_import_airflow(
+        airflow_1_path="airflow.utils.db.create_session",
+        airflow_2_path="airflow.utils.session.create_session",
+    )
+
+
+def lineage_run_id(run_id, task):
+    """
+    Macro function which returns the generated run id for a given task. This
+    can be used to forward the run id from a task to a child run so the job
+    hierarchy is preserved. Invoke as a jinja template, e.g.
+
+    PythonOperator(
+        task_id='render_template',
+        python_callable=my_task_function,
+        op_args=['{{ lineage_run_id(run_id, task) }}'], # lineage_run_id macro invoked
+        provide_context=False,
+        dag=dag
+    )
+
+    :param run_id:
+    :param task:
+    :return:
```

Comment:
Worth adding descriptions to these or removing them completely

Reply:
Removed them.